macros/utils/cross_db_utils/incremental_strategy.sql

```diff
@@ -10,6 +10,10 @@
   {% do return("merge") %}
 {% endmacro %}
 
+{%- macro redshift__get_default_incremental_strategy() %}
+  {% do return("merge") %}
```

Comment:
There is something that broke with `delete+insert` strategy on redshift in fusion (that seemed to be the default in fusion for some reason) - and I didn't think it matters too much so just made it `merge`.

I forgot exactly what was the issue but it didn't seem related to our changes.